src/main/java/projectbookstore/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package projectbookstore.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+@Getter
+@Setter
+public class UserLoginRequestDto {
+    @Email
+    @NotBlank
+    private String email;
+    @NotBlank
+    @Length(min = 8, max = 20)
```

Comment:
It's better not to mix `jakarta.validation` and `hibernate.validator` annotations. Better use `Size`